crates/rust-analyzer/src/semantic_tokens.rs

```diff
@@ -46,7 +46,7 @@ define_semantic_token_types![
     (BRACE, "brace"),
     (BRACKET, "bracket"),
     (BUILTIN_TYPE, "builtinType"),
-    (CHAR_LITERAL, "characterLiteral"),
+    (CHAR, "character"),
```

Comment:
drops the literal suffix here for consistency with the standard lsp token types `string` and `number`.